### PR TITLE
Add the contributing page in community

### PR DESCRIPTION
**Pull request by PragmaTwice**
We add the contributing page named "How to Contribute" in the Community collection.

### CODE REVIEW

File: community/contributing.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+id: contributing
+title: How to Contribute
+---
+
+In order to build an active community to improve Kvrocks, we welcome and are eager to your contributions of all kinds, including but not limited to:
+- suggesting new ideas or feature requests (please refer to the [Community](/community) page);
```
Comment (tisonkun) [inline community/contributing.md:7]:
```suggestion
- suggesting new ideas or feature requests (please refer to the [Community](index.md) page);
```

... for markdown link.

ditto others

**Review by tisonkun — COMMENTED** [review]
Generally looks good.

File: community/contributing.md (hunk 1 above)
Comment (PragmaTwice) [reply community/contributing.md]: done

**Comment by PragmaTwice** [comment]
It is finished and ready for review now :)

File: docs/cluster.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ where
 
 ```shell
 CLUSTERX SETNODES
-  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383
-    07c37dfeb235213a872192d90877d0cd55635b91 127.0.0.1 30004 slave 67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1"
+  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383\n
```
Comment (tisonkun) [inline docs/cluster.md:50]:
```suggestion
  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383 \
```

I'm unsure how redis-cli continues on lines, but may not `\n`? Also, you'd better not pull in unrelated changes in this patch.

File: community/contributing.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+title: How to Contribute
+---
+
+In order to build an active community to improve Kvrocks, we welcome and are eager to your contributions of all kinds, including but not limited to:
+- suggesting new ideas or feature requests (please refer to the [Community](index.md) page);
+- reporting bugs and defects you find (please refer to the [Community](index.md) page); 
+- contributing code changes (whether they are minor typo fixes, improvements, or major feature contributions); 
+- fixing or improving the documentation or the project website.
+
+## Submit Patches to Kvrocks
```
Comment (tisonkun) [inline community/contributing.md:12]:
```suggestion
## Submit patches to Kvrocks
```

use statement capitalization flavor.

**Review by tisonkun — COMMENTED** [review]
Others LGTM. We can improve the content continously.

File: community/contributing.md (hunk 3 above)
Comment (tisonkun) [reply community/contributing.md:12]: ditto others

File: docs/cluster.md (hunk 2 above)
Comment (PragmaTwice) [reply docs/cluster.md]: done

File: community/contributing.md (hunk 3 above)
Comment (PragmaTwice) [reply community/contributing.md]: done

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Merging...
